src/test/java/com/exasol/dbbuilder/dialects/oracle/OracleUserTest.java

```diff
@@ -0,0 +1,49 @@
+package com.exasol.dbbuilder.dialects.oracle;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.collection.IsMapContaining.hasEntry;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.exasol.dbbuilder.dialects.*;
+import com.exasol.dbbuilder.dialects.mysql.MySQLIdentifier;
+import com.exasol.dbbuilder.dialects.mysql.MySqlObjectPrivilege;
+
+@ExtendWith(MockitoExtension.class)
+class OracleUserTest extends AbstractUserTest {
+    @Mock
+    private OracleImmediateDatabaseObjectWriter writerMock;
+
+    @Override
+    protected User createUser(final String name) {
+        return new OracleUser(this.writerMock, MySQLIdentifier.of(name));
+    }
+
+    @Override
+    protected User createUser(final String name, final String password) {
+        return new OracleUser(this.writerMock, MySQLIdentifier.of(name), password);
+    }
+
+    @Override
+    protected DatabaseObjectWriter getWriterMock() {
+        return writerMock;
+    }
+
+    @Test
+    void testGetFullyQualifiedName() {
+        assertThat(new OracleUser(this.writerMock, MySQLIdentifier.of("JOHNDOE")).getFullyQualifiedName(),
+                equalTo("`JOHNDOE`"));
+    }
+
+    @Test
+    void testGetObjectPrivileges(@Mock final DatabaseObject objectMock) {
+        final ObjectPrivilege[] expectedObjectPrivileges = { MySqlObjectPrivilege.INSERT, MySqlObjectPrivilege.DELETE };
+        final User user = new OracleUser(this.writerMock, MySQLIdentifier.of("OBJUSER")) //
+                .grant(objectMock, expectedObjectPrivileges);
```

Comment:
Is it OK to use MySqlObjectPrivilege here? 
Maybe we could add an alias with "Oracle" prefix? 
```suggestion
        final ObjectPrivilege[] expectedObjectPrivileges = { MySqlObjectPrivilege.INSERT, MySqlObjectPrivilege.DELETE };
        final User user = new OracleUser(this.writerMock, OracleIdentifier.of("OBJUSER")) //
                .grant(objectMock, expectedObjectPrivileges);
```

Reply:
Ooops... Thanks for the hint :)